### PR TITLE
remove specificity from datepicker.sass

### PR DESCRIPTION
**Pull request by hippogriffic**
only nest styles where necessary. This will make it easier to overwrite default styles in projects

@incuna/frontend please review

### CODE REVIEW

File: incuna-sass/components/_datepicker.sass [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .datepicker
-    $datepicker-button-size: rem(32px)
+    $datepicker-button-size: rem(32px) !default
```
Comment (pandalion) [inline incuna-sass/components/_datepicker.sass]: what does this do? i forgot :D magic

**Comment by pandalion** [comment]
Good call :+1:

**Comment by grahamgilchrist** [comment]
Nice :)

**Comment by grahamgilchrist** [comment]
I wonder if we should add a comment that this is for use with the angular-bootstrap datepicker? Since there are a lot of datepickers (e.g. jquery etc.) noting this will save someone trying to figure out how to use this?

**Comment by hippogriffic** [comment]
Good idea @grahamgilchrist thanks (and done)

File: incuna-sass/components/_datepicker.sass (hunk 1 above)
Comment (hippogriffic) [inline incuna-sass/components/_datepicker.sass]: makes it easier to overwrite I think, I mostly added it for consistency, we have it on most of the other variables in the library, so we should have it on all really :)

Comment (pandalion) [inline incuna-sass/components/_datepicker.sass]: nice !

**Pull request merged**